src/Store.ts

```diff
@@ -32,14 +34,14 @@ export let defaultStoreName = "<Anonymous-Store>";
  * Store class
  * @public
  */
-export default class Store extends Dispatcher {
+abstract class Store extends Dispatcher {
```

Comment:
`Store` is not abstruct in some test case.
https://github.com/almin/almin/blob/e2daa2fb26d9a3d1d7d215c3d49b8c8ef4430d56/test/Store-test.js#L12
Is it valid in TypeScript?

Of course, `Store` is just abstract in most case without testing, I think.
Should we will modify test code?